includes/class-avatar-store.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace Webmention;
+
+use WP_Comment;
+
+/**
+ * Avatar Store Class
+ *
+ */
+class Avatar_Store {
+	/**
+	 * Initialize the plugin, registering WordPress hooks.
+	 */
+	public static function init() {
+		// Store Avatars Locally
+		add_action( 'comment_post', array( static::class, 'store_avatar' ), 20 );
+		add_action( 'edit_comment', array( static::class, 'store_avatar' ), 20 );
+		// add_action( 'get_comment', array( static::class, 'store_avatar' ), 20 );
```

Comment:
Why is this commented out?

Reply:
Because it led to excessive slowdowns in testing, so I decided to start with just when created or updated. Should have removed the line entirely.